yt/yt/tests/integration/proxies/test_oauth.py

```diff
@@ -19,7 +19,11 @@ def auth_config(port):
             "user_info_endpoint": "user_info",
             "user_info_login_field": "login",
             "user_info_subject_field": "sub",
-            "user_info_error_field": "error"
+            "user_info_error_field": "error",
+            "login_transformation": {
+                "match_pattern": "(.*)@(.*)\\.(.*)",
+                "replacement": "\\1-\\2",
```

Comment:
```suggestion
                "match_pattern": r'(.*)@(.*)\.(.*)',
                "replacement": r'\1-\2',
```
You may want to use python raw strings to avoid double escaping so regexes would look like ones user would put in text config.

Reply:
Suggested changes, much wow.